src/insane/mcmc_cfbd.jl

```diff
@@ -1,9 +1,9 @@
 #=
 
-constant fossilized birth-death MCMC wrapper
+constant fossilized birth-death MCMC using graft and prune
```

Comment:
I guess this is not using graft and prune but forward simulation?

Reply:
Right I changed to "using forward simulation"